mods/ra/maps/soviet-05/AI.lua

```diff
@@ -0,0 +1,143 @@
+BaseBuildings =
+{
+	{ "powr", CVec.New(3, -2), 300 },
+	{ "tent", CVec.New(0, 4), 400 },
+	{ "hbox", CVec.New(3, 6), 600 },
+	{ "proc", CVec.New(4, 2), 1400 },
+	{ "powr", CVec.New(5, -3), 300 },
+	{ "weap", CVec.New(-5, 3), 2000 },
+	{ "hbox", CVec.New(-6, 5), 600 },
+	{ "gun", CVec.New(0, 8), 600 },
+	{ "gun", CVec.New(-4, 7), 600 },
+	{ "powr", CVec.New(-4, -3), 300 },
+	{ "proc", CVec.New(-9, 1), 1400 },
+	{ "powr", CVec.New(-8, -2), 300 },
+	{ "silo", CVec.New(6, 0), 150 },
+	{ "agun", CVec.New(-3, 0), 800 },
+	{ "powr", CVec.New(-6, -2), 300 },
+	{ "agun", CVec.New(4, 1), 800 },
+	{ "gun", CVec.New(-9, 5), 600 },
+	{ "gun", CVec.New(-2, -3), 600 },
+	{ "powr", CVec.New(4, 6), 300 },
+	{ "gun", CVec.New(3, -6), 600 },
+	{ "hbox", CVec.New(3, -4), 600 },
+	{ "gun", CVec.New(2, 3), 600 }
+}
+
+BuildBase = function()
+
```

Comment:
It is also known as a carriage return. It is the space between one line and another.